docs/docs/learn/spaceward/tutorial-warden-metamask-snap.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 10
+---
+
+# Tutorial - Warden MetaMask Snap
+
+Now, with the groundbreaking MetaMask Snap feature, you can seamlessly integrate your Test Warden Protocol generated Ethereum addresses directly into MetaMask Flask, ushering in a new era of convenience and accessibility - you will be able to see those addresses tagged “Warden Protocol” in your MetaMask Flask Wallet.
+
+### 1. Requirements
+
+As we are in testnet ensure you have MetaMask Flask installed, the developer-centric version.. Once the final version is released, it will be available through the standard orange MetaMask extension. We will need to be whitelisted to have it in the original MetaMask, pending security audit. 
```

Comment:
```suggestion
As we are in testnet ensure you have MetaMask Flask installed, the developer-centric version. Once the final version is released, it will be available through the standard orange MetaMask extension. We will need to be whitelisted to have it in the original MetaMask, pending security audit. 
```